tests/integration_test.rs

```diff
@@ -382,3 +383,129 @@ async fn test_oci_manifest_capability(
+) {
+    let (callback_handler_shutdown_channel_tx, callback_handler_shutdown_channel_rx) =
+        oneshot::channel();
+    let callback_builder = CallbackHandlerBuilder::new(callback_handler_shutdown_channel_rx);
+    let mut callback_handler = callback_builder
+        .build()
+        .await
+        .expect("cannot build callback handler");
+    let callback_handler_channel = callback_handler.sender_channel();
+
+    tokio::spawn(async move {
+        callback_handler.loop_eval().await;
+    });
+    let (tx, rx) = oneshot::channel::<Result<CallbackResponse>>();
+    let req = CallbackRequest {
+        request: CallbackRequestType::OciManifestAndConfig {
+            image: policy_uri.to_owned(),
+        },
+        response_channel: tx,
+    };
+
+    let eval_ctx = EvaluationContext {
+        policy_id: "test".to_owned(),
+        callback_channel: Some(callback_handler_channel),
+        ctx_aware_resources_allow_list: Default::default(),
+    };
+
+    let cb_channel: mpsc::Sender<CallbackRequest> = eval_ctx
+        .callback_channel
+        .clone()
+        .expect("missing callback channel");
+    assert!(cb_channel.try_send(req).is_ok());
+
+    // wait for the response
+    match rx.await {
+        Ok(msg) => match msg {
+            Ok(response_raw) => {
+                let response: serde_json::Value =
+                    serde_json::from_slice(&response_raw.payload).unwrap();
+                println!("{:?}", response);
+                if let Some(expected_config_json) = expected_config {
+                    assert!(response
+                        .get("config")
+                        .unwrap()
+                        .get("config")
+                        .unwrap()
+                        .eq(&expected_config_json));
+                } else {
+                    assert!(response
+                        .get("config")
+                        .unwrap()
+                        .as_object()
+                        .unwrap()
+                        .is_empty());
+                }
+                let manifest: OciImageManifest =
+                    serde_json::from_value(response.get("manifest").unwrap().clone())
+                        .expect("cannot deserialize manifest");
+                // The OciImageManifest does not implement the Eq trait. Therefore, we need to
+                // compare the fields one by one. Thus, let's checks some of them only. We do not
+                // need to validate all of them here.
+                assert_eq!(manifest.media_type, expected_manifest.media_type);
+                assert_eq!(manifest.schema_version, expected_manifest.schema_version);
+                assert_eq!(
+                    manifest.config.to_string(),
+                    expected_manifest.config.to_string()
+                );
+            }
+            Err(e) => {
+                panic!("{}", e)
+            }
+        },
+        Err(e) => {
+            panic!("{}", e)
+        }
+    }
```

Comment:
Instead of using `match` we could simply expect/unwrap twice
e.g.

```rust
    let response_raw = rx
        .await
        .expect("cannot receive response")
        .expect("cannot get response");
```
The same applies for the `test_oci_manifest_capability` test in this file, could we fix this one as well?